keras_nlp/metrics/bleu_test.py

```diff
@@ -142,12 +144,10 @@ def test_different_order(self):
 
     def test_model_compile(self):
         inputs = keras.Input(shape=(), dtype="string")
-        outputs = tf.identity(inputs)
-        model = keras.Model(inputs, outputs)
-
+        model = keras.Model(inputs, inputs)
```

Comment:
IMO it's kinda nice to have the explicit version where we have different outputs, even if we just use `ops.identity`.

Totally an optional nit

Reply:
Similar answer as below. `ops.identity` will fail on `tf.keras`'s `KerasTensor` (they aren't designed to interop), and `tf.identity` will fail on `keras_core`'s `KerasTensor` ([see these lines](https://github.com/keras-team/keras-core/blob/ce9b511a063f0be3988ba34accc2e07dc1812ab9/keras_core/backend/common/keras_tensor.py#L87-L108)).

But I just realized we can use `keras.layers.Identity` in all cases, will make that change.